versions/1/changelog.yml

```diff
@@ -9,6 +9,9 @@
     link: https://github.com/elastic/package-spec/pull/265
 - version: 1.4.0
   changes:
+  - description: Support platinum license type.
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/267
```

Comment:
Please move the changelog entry to the version `1.4.1-next` above.

Reply:
Updated in 0aa6454b67bd7d760b7dcc644e91050de50edaa2